kodilla-stream/src/main/java/com/kodilla/stream/StreamMain.java

```diff
@@ -1,7 +1,14 @@
 package com.kodilla.stream;
 
+import com.kodilla.stream.beautifier.PoemBeautifier;
+
 public class StreamMain {
     public static void main(String[] args) {
-        System.out.println("Welcome to module 7 - Stream");
+        PoemBeautifier poemBeautifier = new PoemBeautifier();
+
+        poemBeautifier.beautify("Litwo, ojczyzno moja", (string -> "***" + string + "***"));
+        poemBeautifier.beautify("Ty jesteś jak zdrowie", (string -> string.toUpperCase()));
```

Comment:
generalnie trzeba by bylo jeszcze dodac obsluzenie sytuacji, w ktorej przekazujemy null, bo nam poleci NPE (Null Pointer Exception).
„string” to niezbyt dobra nazwa zmiennej/argumentu - lepiej dac str